.hintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": [
+    "development"
+  ],
```

Comment:
why is this file committed ? please remove this file from commit history

Reply:
Yes sir i have removed.